data/hooks/conf_regen/25-dovecot

```diff
@@ -27,6 +27,13 @@ do_pre_regen() {
       "${dovecot_dir}/dovecot.conf"
   fi
 
+  # Allow to configure pop3 protocol support
+  export pop3_enabled="$(yunohost settings get 'pop3.enabled')"
+  if [ ! -z "${pop3_enabled}" ]; then
+    ynh_package_install dovecot-pop3d
+    yunohost firewall allow TCP 995
```

Comment:
Well, maybe we only wan to enable it locally. By example for the discourse app. I purpose to keep closed the firewall and the use can open it manually if he want.

Reply:
Ah, that's a good point! And it gets rid of the issue of having to configure the firewall for it.

OK, I'll remove this line. I might document that somewhere regarding the port management.